lib/stdlib/doc/src/orddict.xml

```diff
@@ -81,6 +99,22 @@
           An exception is generated if the initial value associated with
           <c><anno>Key</anno></c> is not a list of values.</p>
         <p>See also section <seeerl marker="#notes">Notes</seeerl>.</p>
+          <p><em>Example 1:</em></p>
+          <pre>
+1> <input>OrdDict1 = orddict:from_list([{a, 1}, {b, 2}]).</input>
+[{a,1},{b,2}]
+2> <input>OrdDict2 = orddict:append_list(c, [3, 4], OrdDict1).</input>
+[{a,1},{b,2},{c,[3, 4]}]
+3> <input>orddict:fetch(c, OrdDict2).</input>
+[3,4]
+</pre>
+          <p><em>Example 2:</em></p>
```

Comment:
Here is an instance where (in my opinion) having two examples only makes it harder to read. I suggest:

```
1> OrdDict1 = orddict:from_list([{a, 1}, {b, 2}]).
[{a,1},{b,2}]
2> orddict:fetch(a, OrdDict1).
1
3> orddict:fetch(missing, OrdDict1).
** exception error: no function clause matching orddict:fetch(missing,[])
```